README.md

```diff
@@ -94,13 +94,13 @@ Seamless-immutable supports both static and instance syntaxes:
 ```
```

Comment:
I've updated the PR but this comment is on a the close comment so it didn't hide it :)